trace-forward/trace-forward.cabal

```diff
@@ -0,0 +1,99 @@
+
+common common-options
+  build-depends:       base >=4.12 && <5
+
+  ghc-options:         -Wall
+                       -Wcompat
+                       -Widentities
+                       -Wincomplete-uni-patterns
+                       -Wincomplete-record-updates
+  if impl(ghc >= 8.0)
+    ghc-options:       -Wredundant-constraints
+  if impl(ghc >= 8.2)
+    ghc-options:       -fhide-source-paths
+  if impl(ghc >= 8.4)
+    ghc-options:       -Wmissing-export-lists
+                       -Wpartial-fields
+
+  default-language:    Haskell2010
+
+library
+  import:              common-options
+  hs-source-dirs:      src
+
+  exposed-modules:     Trace.Forward.Acceptor
+                       Trace.Forward.Configuration
+                       Trace.Forward.Forwarder
+                       Trace.Forward.Queue
+
+                       Trace.Forward.Network.Acceptor
+                       Trace.Forward.Network.Forwarder
+
+                       Trace.Forward.Protocol.Acceptor
+                       Trace.Forward.Protocol.Codec
+                       Trace.Forward.Protocol.Forwarder
+                       Trace.Forward.Protocol.Limits
+                       Trace.Forward.Protocol.Type
+
+  build-depends:         async
+                       , bytestring
+                       , cborg
+                       , contra-tracer
+                       , extra
+                       , io-classes
+                       , network
+                       , ouroboros-network-framework
+                       , serialise
+                       , stm
+                       , text
+                       , time
+                       , typed-protocols
+
+test-suite test
+  type:                exitcode-stdio-1.0
+  main-is:             Main.hs
+  hs-source-dirs:      test
+
+  other-modules:       Test.Trace.Forward.Protocol.Codec
+                       Test.Trace.Forward.Protocol.Tests
+                       Test.Trace.Forward.Protocol.TraceItem
+                       Test.Trace.Forward.Demo.Configs
+                       Test.Trace.Forward.Demo.Tests
+
+  build-depends:         base
+                       , bytestring
+                       , contra-tracer
+                       , directory
+                       , filepath
+                       , io-classes
+                       , ouroboros-network-framework
+                       , trace-forward
+                       , QuickCheck
+                       , serialise
+                       , stm
+                       , tasty
+                       , tasty-quickcheck
+                       , typed-protocols
+                       , typed-protocols-examples
+                       , text
+                       , time
+
+  default-language:    Haskell2010
+  ghc-options:         -rtsopts
+                       -Wall
+                       -Wno-unticked-promoted-constructors
+                       -Wno-orphans
+                       -threaded
```

Comment:
Why not the other useful options as in the `common-options` library?

Reply:
Fixed.